db/seeds/schemas/add_agfs_fee_scheme_12.rb

```diff
@@ -166,7 +156,7 @@ def copy_scheme_11_offences
         Offence.transaction do
           agfs_scheme_eleven_offences.each do |offence|
             if pretending?
-              puts "#{offence.unique_code} => #{offence.unique_code.sub('~11','~12')}".yellow
+              puts "[WOULD-COPY] " + "#{offence.unique_code} => #{offence.unique_code.sub('~11','~12')}".yellow
```

Comment:
@jsugarman is this static string `[WOULD-COPY] ` required?

Reply:
its just so its just to make it a different colour
![Screen Shot 2020-09-18 at 13 51 27](https://user-images.githubusercontent.com/7016425/93599503-234f8700-f9b6-11ea-911e-1697521db351.png)